assignment/src/com/assignment3/fileInputOutput/Main.java

```diff
@@ -0,0 +1,13 @@
+package com.company;
+
+import java.io.IOException;
+
+public class Main {
+
+    public static void main(String[] args) throws IOException {
+        //main class object
+        InputAndOutputStreams streams =new InputAndOutputStreams();
+        //assigning the methods
+        streams.filesExchange();
```

Comment:
Need  to display content of Target file.

Reply:
fix